scripts/commands/makepearlsack.lua

```diff
@@ -0,0 +1,30 @@
+-----------------------------------
+-- func: makepearlsack
```

Comment:
not sure about that command name, but the code is good.

I used `!givels` and `!getls` (the latter self only for a specific ls) on my version

Reply:
Renamed and made it targetable to you can give players a pearl directly.